apps/frontend/src/components/fap/MeetingComponents/FapMeetingInstrumentsTable.tsx

```diff
@@ -202,6 +224,16 @@ const FapMeetingInstrumentsTable = ({
           debounceInterval: 400,
         }}
       />
+
+      {updateInstrumentTime && selectedCall && (
+        <FapUpdateInstrumentTime
+          open={!!updateInstrumentTime}
```

Comment:
I think in this case you might not need to pass the `open` property from outside if you check `updateInstrumentTime` and `selectedCall` already above. Otherwise I think it might be smarter to remove the check above and add something like: `open={!!updateInstrumentTime && !!selectedCall}`